indices/split_pairwise_network.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import os
+import pickle as pkl
+import re
+
+import networkx as nx
+
+from utils import parse_mesh_headings
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('in_file',
+                        help='The file to be split',
+                        default='data/networks')
```

Comment:
This default isn't a file, though, is it? It looks like a directory.

I know you're providing this argument when you call this script in `snakemake` so it probably doesn't matter too much.

Reply:
Good catch! This script used to take in a directory and I updated the argument name but didn't remove the default